src/sqlparser/tests/testdata/show.yaml

```diff
@@ -11,7 +11,7 @@
 - input: SHOW TABLES FROM t
   formatted_sql: SHOW TABLES FROM t
   formatted_ast: 'ShowObjects { object: Table { schema: Some(Ident { value: "t", quote_style: None }) }, filter: None }'
-- input: SHOW TABLES FROM t LIKE "t%"
+- input: SHOW TABLES FROM t LIKE 't%'
```

Comment:
Strictly speaking this fix is a breaking change. But it was wrong syntax and our documentation has been using single quotes correctly. The impact shall be negligible.